elastic/beats
# [Agent] Make default config OS agnostic

## What does this PR do?

PR changes path of `install` and `download` directories in default config to be OS agnostic by using `path.data`.

## Why is it important?

It simplifies first run experience when running agent by removing the need to change configuration values which are configured to non existent paths.

## Checklist


## How to test this PR locally

You need to build  and package agent to contain `data/beats` subdirectory containing filebeat and metricbeat archives (tar.gz on linux and darwin). 

Second step is just running agent and verifying that metricbeat was installed to `data/install` and is successfully started
